### PR TITLE
refactor select page

### PR DESCRIPTION
**Pull request by dworthen**
- update to latest dwc and @essex utility packages
- pull out table header commands to top level page commands
  - select columns, subject ID, sensitive zeros, multi value columns
- add select menu callbacks for page commands
  - support selecting all and none options
- default to using no columns for sensitive data.
have users select which columns to use as first step
on select page
- add multi value page command placeholder
  - add multi value recoil state
- show table preview once columns have been selected
- show table statistics (columns, rows and unique values)

### CODE REVIEW

File: packages/webapp/src/pages/Select/InfoBar/InfoBar.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+					acc.combinations *= colStats.stats?.distinct ?? 1
+				}
+
+				return acc
+			},
+			{ count: 0, unique: 0, combinations: 1 },
+		)
+
+		if (data.count) {
+			return `${data.count} columns, ${rowCount} rows, and ${data.unique} unique values.` // with ${data.combinations} maximum possible combimations`
```
Comment (rracanicci) [inline packages/webapp/src/pages/Select/InfoBar/InfoBar.tsx:33]: Is there an extra "," here after rows?

File: packages/webapp/src/pages/Select/SelectCommands/SensitiveZeros/SensitiveZeros.tsx [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+				headers: newHeaders,
+			}))
+		},
+		[sensitiveContent, setSensitiveContent, columnsWithZeros],
+	)
+
+	const list = useMemo(() => {
+		return (
+			sensitiveContent.headers
+				// .filter(header => header.use && columnsWithZeros?.includes(header.name))
```
Comment (rracanicci) [inline packages/webapp/src/pages/Select/SelectCommands/SensitiveZeros/SensitiveZeros.tsx:53]: Maybe we can remove this?

**Review by rracanicci — CHANGES_REQUESTED** [review]
Looking extremely good. Just suggesting some minor changes.

File: packages/webapp/src/pages/Select/DataSelect/DataSelect.tsx [hunk 3]
```diff
@@ -37,6 +37,9 @@ export const DataSelect: FC = memo(function DataSelect() {
 	}
 
 	const visibleColumns = useVisibleColumnNames(sensitiveContent)
+	// const visibleColumns = useMemo(() => {
```
[the remark this answers is not in the document]
Comment (dworthen) [reply packages/webapp/src/pages/Select/DataSelect/DataSelect.tsx]: This component is not being used anymore and will be removed in a future PR. There are a few things I still need to take from the component which is why I have not removed it yet.

**Pull request merged**